Makefile

```
@@ -73,6 +73,10 @@ push: ## Push the Docker image to the registry
 dependencies: ## Install dependencies required by the application
 	npm install
 
+.PHONY: dependencies
+ci-dependencies: ## Install dependencies required by the application in CI
```

I wonder if there would be a better name for this task. I’ve parsed `npm ci` documentation and we could use

```suggestion
dependencies-strict: ## Install dependencies required by the application from a clean slate
```

What do you think? I’m trying to avoid including a more generic term than “ci” in the name but I might be wrong 😄

---

I think the description is more accurate but "strict" doesn’t really reflect that IMO. I’m not sure how to express that it’s installing from a clean slate 🤔

---

@remi it’s longer but what do you think of `dependencies-clean-slate`?

---

😅 Yeah, it’s not that great, is it? I think we could stick with `ci-dependencies` for now 😁